MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -712,9 +704,45 @@ public bool SupportsSubtitleConversionTo(string toCodec)
                 || string.Equals(codecTag, "dvhe", StringComparison.OrdinalIgnoreCase)
                 || string.Equals(codecTag, "dav1", StringComparison.OrdinalIgnoreCase))
             {
+                isDoVi = true;
+            }
+
+            var colorTransfer = ColorTransfer;
+
+            if (string.Equals(colorTransfer, "smpte2084", StringComparison.OrdinalIgnoreCase))
+            {
+                isHDR10 = true;
+            }
+
+            if (string.Equals(colorTransfer, "arib-std-b67", StringComparison.OrdinalIgnoreCase))
+            {
+                isHLG = true;
+            }
+
+            if (isDoVi)
+            {
+                if (isHDR10)
+                {
+                    return (VideoRange.HDR, VideoRangeType.DolbyVisionWithHDR10Fallback);
+                }
+                else if (isHLG)
+                {
+                    return (VideoRange.HDR, VideoRangeType.DolbyVisionWithHLGFallback);
+                }
```

Comment:
IMO this is not that reliable. We have `dvBlCompatId` that can be used to identify the DOVI fallback types.

https://github.com/jellyfin/jellyfin/blob/8f2829e1aa696d37d338fceed2ea1d9edf1ca824/MediaBrowser.Model/Entities/MediaStream.cs#L206-L208